cmd/config/internal/commands/source.go

```diff
@@ -29,6 +29,8 @@ func GetSourceRunner(name string) *SourceRunner {
 		"output using this format.")
 	c.Flags().StringVar(&r.FunctionConfig, "function-config", "",
 		"path to function config.")
+	c.Flags().StringSliceVar(&r.InputPattern, "input-pattern", kio.DefaultMatch,
```

Comment:
How do we signal stdin to accept json files ?

Reply:
Tests for source ? They look similar to [this](https://github.com/kubernetes-sigs/kustomize/pull/2546/files#diff-5bd68abc0972422aef6be3cf0eb26b1aR138) I suppose? But stdin might not work.

Reply:
JSON parses as YAML -- we don't need to signal anything for STDIN.

Reply:
Yes, please add a test for source.  The intent of this PR was to give you direction, not to be the solution itself.